### ISSUE TITLE
Shell clear command

### ISSUE THREAD

**Issue by pardom-zz** [mttkay/replicant#32]

**Comment by mttkay** [comment]
Could provide more info as to what this should achieve?

**Comment by pardom-zz** [comment]
Yeah, sorry I didn't add a description. I was looking for a `clear` command like in shell. It clears the terminal window.

Another thing I keep trying to do is use the history by pressing the up key, which obviously doesn't work.

**Comment by mttkay** [comment]
CMD+K? :-) It's built in to bash/zsh

**Comment by pardom-zz** [comment]
Oh, cool. I've always just used `clear`. What about bash history? Is it possible to add that?
